packages/jest/src/index.js

```diff
@@ -21,9 +21,9 @@ function getFinalPath(path) {
 }
 
 function normalizeJestOptions(opts, neutrino, usingBabel) {
-  const mediaNames = `\\.(${mediaExtensions.join('|')})`;
-  const styleNames = `\\.(${['css', 'less', 'sass', 'scss'].join('|')})`;
-  const jsNames = neutrino.regexFromExtensions(['js', 'jsx']);
+  const mediaNames = `\\.(${mediaExtensions.join('|')})$`;
+  const styleNames = `\\.(${['css', 'less', 'sass', 'scss'].join('|')})$`;
+  const jsNames = `\\.(${['js', 'jsx'].join('|')})$`;
```

Comment:
We still want to use the stored Neutrino extensions, so probably something like:

```js
`\\.(${[...neutrino.options.extensions].join('|')})$`;
```

Reply:
I think we should also add a comment explaining why we're not using `regexFromExtensions()` :-)